packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -16,6 +16,7 @@
     @keydown.up.stop.prevent="showDropdown()"
     @keydown.down.stop.prevent="showDropdown()"
     @keydown.space.stop.prevent="showDropdown()"
+    @keydown.enter.stop.prevent="showDropdown()"
```

Comment:
The native html `select` element has no such option. Do we want to do it?

Reply:
Native select option selection also works for `enter`.

Reply:
Enter usually used for submit action in form. I would rather just use space to open select. Pressing enter on any component could submit my form.

Also, up and down buttons often used not for dropdown opening, but switching between options without opening dropdown.